enos/ansible/plugins/callback/influxdb_events.py

```diff
@@ -3,134 +3,189 @@
+        """Add a new event in the list"""
+
+        # Add Kolla-related information in tags
+        fields['tags'] = '%s %s %s %s' % (fields['tags'],
+                self.extra_vars['kolla_ref'],
+                self.extra_vars.get('kolla_base_distro') or \
+                        self.extra_vars['kolla']['kolla_base_distro'],
+                self.extra_vars.get('kolla_install_type') or \
+                        self.extra_vars['kolla']['kolla_install_type']
+        )
+
+        # Add the OpenStack release tag if available
+        if self.openstack_release:
+            fields['tags'] = '%s %s' % (fields['tags'], self.openstack_release)
+        # Add the OpenStack region name if available
+        if self.openstack_region_name:
+            fields['tags'] = '%s %s' % (fields['tags'],
+                    self.openstack_region_name)
+
         current_time = datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%S.%f')
-        event = [
-            {
-                'time': current_time,
-                'measurement': 'events',
-                'tags': tags,
-                'fields': fields
-            }
-        ]
+
+        event = {
+            'time': current_time,
+            'measurement': 'events',
+            'fields': fields
+        }
+
         self.events.append(event)
 
+
     def v2_playbook_on_start(self, playbook):
-        """
-        Logs each playbook start
-        """
-        self.username = pwd.getpwuid(os.getuid()).pw_name
-        self.hostname = socket.gethostname()
-        self.playbook = playbook
-        self.playbook_name = os.path.basename(playbook._file_name)
-        self.playbook_basedir = playbook._basedir
+        """Log each starting playbook"""
+
+        playbook_name = os.path.basename(playbook._file_name)
+
+        variable_manager = playbook._entries[0]._variable_manager
+        self.extra_vars = variable_manager.extra_vars
 
         fields = {
-            'text': 'Playbook %s started on %s by %s' %
-               (self.playbook_name, self.hostname, self.username)
+            'tags': 'playbook %s' % self.username,
+            'text': playbook_name,
+            'type': 'playbook',
+            'title': 'Playbook Started'
         }
-        self.report_event('playbook-start', fields)
+
+        self.report_event(fields)
+
 
     def v2_playbook_on_play_start(self, play):
-        """
-        Logs each play start
-        """
+        """Log each starting play"""
+
+        host = play._variable_manager._hostvars.keys()[0]
+        host_vars = play._variable_manager._hostvars[str(host)]
+
+        # If not set, fetch the value of `enable_monitoring` from either the
+        # config file, or from Ansible playbook variables.
+        # NOTE(dpertin) before gathering facts `enable_monitoring` is not
+        # provided as a playbook variable, thus we consider it is enabled by
+        # default.
+        enable_monitoring = self.extra_vars.get('enable_monitoring') \
```

Comment:
@msimonin I have removed `extra_vars` since information can be found in `host_vars`.

However, `enable_monitoring` is not available in `host_vars` with `kolla_ansible` when it is not set in `reservation.yml` since it is an Enos variable.

Thus, this new proposition checks if `enable_monitoring` is set in `host_vars`:
* if `enable_monitoring` is not set in `reservation.yml`, it will be considered `True`,
* if `enable_monitoring` is set in `reservation.yml`, it will be available anytime in `host_vars`.